Example/Controllers/ViewController.swift

```diff
@@ -46,6 +48,11 @@ class ViewController: RATViewControllerWithTableView {
         self.tableView.refreshControl = refreshControl
         locationManager.delegate = self
         self.pageName = MASDKLocale.localize("demo.app.rat.page.name.home")
+        miniAppIAPModule = MiniAppIAPModule()
+        guard let inAppPurchaseModule = miniAppIAPModule else {
+            return
+        }
+        SKPaymentQueue.default().add(inAppPurchaseModule)
```

Comment:
I think because the `miniAppModule` will be set anyways this could be written with an if which would be a bit more readable like below wdyt?

```suggestion
        if miniAppIAPModule == nil {
            let module = MiniAppIAPModule()
            self.miniAppIAPModule = module
            SKPaymentQueue.default().add(module)
        }
```

Reply:
Yep. Looks good, updated code now